appservice/src/main/java/nu/yona/server/BruteForceAttemptService.java

```diff
@@ -1,136 +0,0 @@
-/*******************************************************************************
```

Comment:
It was used at two places: overwrite account and confirm number. Both are now replaced by the ``ConfirmationCode`` entity.